corehq/apps/domain/views/releases.py

```diff
@@ -78,13 +79,20 @@ def activate_release_restriction(request, domain, restriction_id):
     if not toggles.MANAGE_RELEASES_PER_LOCATION.enabled_for_request(request):
         return HttpResponseForbidden()
     latest_enabled_app_release = LatestEnabledAppReleases.objects.get(id=restriction_id, domain=domain)
-    latest_enabled_app_release.activate()
-    response_content = {
-        'id': restriction_id,
-        'success': True,
-        'activated_on': (datetime.strftime(latest_enabled_app_release.activated_on, '%Y-%m-%d %H:%M:%S')
-                         if latest_enabled_app_release.activated_on else None),
-        'deactivated_on': (datetime.strftime(latest_enabled_app_release.deactivated_on, '%Y-%m-%d %H:%M:%S')
-                           if latest_enabled_app_release.deactivated_on else None),
-    }
+    try:
+        latest_enabled_app_release.full_clean()
+    except ValidationError as e:
+        response_content = {
+            'message': ','.join(e.messages)
```

Comment:
Seems like this should return a 400 response code or something

Reply:
we don't set `success` and then it's handled at ajax. If that's not a good practice and i should change it to 400 and then change the javascript code instead, don't mind doing that.

Reply:
I don't think it's a huge deal - whichever feels cleaner.  The main advantage of using response codes is that other tools will know what the response means, such as your browser console or the jquery ajax library (it has hooks for `error` and `statusCode` callbacks).  I'd probably default to sticking with standards, but wouldn't be foolishly consistent if it ends up being messier than doing something custom.